packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx

```diff
@@ -11,4 +13,9 @@ const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
 const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
 const displays = [display1, display2, display3, display4, display5, display6];
 
-export default () => <ItemView displays={displays} layout="twoColumns" />;
+export default () => (
+  <div>
+    <JSXDoc />
```

Comment:
Why can't we move these example to this file?

Reply:
I think this documentation is redundant as the same has been used for creating the example.